iidm/iidm-api/src/main/java/com/powsybl/iidm/network/DanglingLine.java

```diff
@@ -98,8 +101,21 @@
      */
     String getUcteXnodeCode();
 
-    CurrentLimits getCurrentLimits();
-
-    CurrentLimitsAdder newCurrentLimits();
-
+    /**
+     * @deprecated Use {@link #getOperationalLimits(LimitType, Class)} instead.
+     */
+    @Deprecated
+    default CurrentLimits getCurrentLimits() {
+        return getOperationalLimits(LimitType.CURRENT, CurrentLimits.class);
+    }
+
```

Comment:
Same remark, how do I access to apparent power limits and active power limits ?

Reply:
Indeed you just leave ````getCurrentLimits()```` for compatibility but you will remove it ?

Reply:
Yes, this is it.